src/main/java/com/arpnetworking/metrics/mad/parsers/CollectdJsonToRecordParser.java

```diff
@@ -280,7 +281,7 @@ private CollectdRecord(final Builder builder) {
          * {@link com.arpnetworking.commons.builder.Builder} implementation for
          * {@link CollectdRecord}.
          */
-        public static final class Builder extends ThreadLocalBuilder<CollectdRecord> {
+        public static final class Builder extends OvalBuilder<CollectdRecord> {
```

Comment:
unclear why this changed

Reply:
The parser is instantiated in a way that does not use the ThreadLocalBuilder correctly resulting in warnings in the log. It's not related to this PR; just discovered in the testing of this PR.

In general, TLB should be used in areas where objection creation velocity is high. That is not the case with Parsers or Sinks. Therefore, in these cases it is sufficient to use OB.

Reply:
Makes sense, thanks for clarifiying!

Reply:
I think you changed the wrong one to OvalBuilder here.  I don't think the CollectdJsonToRecordParser class has a builder.